Challenge1/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const CREATE_AGENT_ABI = "function createAgent(uint256 agentId,address ,string metadata,uint256[] chainIds)";
+export const UPDATE_AGENT_ABI = "function updateAgent(uint256 agentId,string metadata,uint256[] chainIds)";
+export const OTHER_FUNCTION_ABI = "function isRegistered(uint256 agentId) returns (bool)";
```

Comment:
Since this one isn't used in production, I think it would be best to be moved to the test file. Then you can use it as you currently are. 🙂